apps/jsonrpc2/mix.exs

```diff
@@ -45,7 +43,8 @@ defmodule JSONRPC2.Mixfile do
       {:credo, "~> 1.0.0-rc1", only: [:dev, :test], runtime: false},
       {:dialyxir, "~> 1.0.0-rc.4", only: [:dev, :test], runtime: false},
       {:hackney, "~> 1.6", only: [:test]},
-      {:websockex, git: "https://github.com/InoMurko/websockex.git", branch: "master", only: [:test]},
+      {:websockex,
+       git: "https://github.com/InoMurko/websockex.git", branch: "master", only: [:test]},
```

Comment:
that's bad. why are we using a fork?

Reply:
because the original has dialyzier issues and referencing old dependencies (cowboy 1, plug 1).
I fixed that and send them a PR https://github.com/Azolo/websockex/pull/63/files

Reply:
what if we fork mine into mana repo?

Reply:
@InoMurko it would be nice